server/src/thread/model.rs

```diff
@@ -39,7 +39,7 @@ impl Threads {
 
         let result = sqlx::query!(
             r#"
-                SELECT comment.id
+                SELECT comment.id, comment.date
```

Comment:
In our current understanding `comment.date` shall be the date when the comment was created and it shall not be updated for new comments

Reply:
Use a SQL statement like `select comment.id, comment.date, MAX(answer.id) as most_recent_answer_id from comment join uuid on comment.uuid_id = uuid.id join comment answer on answer.parent_id = comment.id group by comment.id limit 10;` -> Order by `most_recent_answer_id` then

Reply:
Thanks for your hint in slack 😃